skyportal/handlers/api/galaxy.py

```diff
@@ -121,12 +123,98 @@ def get(self, catalog_name=None):
         """
 
         catalog_name = self.get_query_argument("catalog_name", None)
+        localization_dateobs = self.get_query_argument("localizationDateobs", None)
+        localization_name = self.get_query_argument("localizationName", None)
+        localization_cumprob = self.get_query_argument("localizationCumprob", 0.95)
+        includeGeojson = self.get_query_argument("includeGeojson", False)
```

Comment:
none of these variable are described in the docstring for this get method.

Reply:
added.